godot_communicator.py

```diff
@@ -50,15 +54,17 @@ def godot_sender(udp_ip, udp_port, track):
             continue
         except socket.timeout:
             # drive car. sleep amount is inversely dependant of velocity
-            velocity = calculate.velocity(velocity, 0)
+            velocity = calculate.velocity(velocity, time_stamp, throttle_force=0)
```

Comment:
There are 2 separate functions modifying this `time_stamp` value like 10+ times a second. I can almost guarantee you that you'll be passing an unexpected value frequently.

Reply:
I just realized there is another issue with the current code, which has been put under a microscope due to the time_stamp variable.
**Scenario**
1. Space bar is pressed (held down)
2. new velocity is calculated
3. sender function runs
4. new velocity is calculated again
5. velocity after 2 calculations is sent <-- expected behavior?

This will need to be discussed as well on how to fix this or if we accept it.